specs/flashtestations.md

```diff
@@ -341,87 +359,97 @@ A TEE's workload identity is derived from a combination of its measurement regis
 
 1. **Storing addresses** that have been validated through attestation
 2. **Associating addresses** with their specific workload identity
-3. **Storing attestation quotes** for future verification
-4. **Providing efficient lookup** capabilities to verify if an address is authorized
+3. **Storing attestation quotes** for future verification and revocation
+4. **Providing efficient lookup** capabilities to verify if an address is authorized for a particular workloadID
 
 The registry operates on these key abstractions:
 
-1. **Workload Identity (`workloadId`)**: A 32-byte hash derived from TDX measurement registers (as defined in [Workload Identity Derivation](#workload-identity-derivation)) that uniquely identifies a specific piece of code running in a TDX environment. This serves as the primary namespace under which addresses are stored.
+1. **TEE-controlled address**: The address extracted from the attestation's report data field ([TDReport.ReportData](#tdreport)), whose private key was generated inside the TEE and is used to interact with onchain contracts.
 
-2. **Attestation Quote**: The raw attestation data provided during registration that contains the cryptographic proof of the TEE's state. This quote is stored alongside the address for later verification.
+2. **Parsed Attestation**: A struct containing the verified and attested data. It contains the quote in its raw form as well as extracted values which are often used and required such as the workloadId.
 
-3. **Ethereum Address**: The public key extracted from the attestation's report data field ([TDReport.ReportData](#tdreport)), which will be used to interact with onchain contracts.
+2.1 **Attestation Quote**: The raw attestation data provided during registration that contains the cryptographic proof of the TEE's state. This quote is stored alongside the address for later verification and revocation.
 
-### Key Relationship Model
+2.2 **Workload Identity (`workloadId`)**: A 32-byte hash derived from TDX measurement registers (as defined in [Workload Identity Derivation](#workload-identity-derivation)) that uniquely identifies a specific piece of code running in a TDX environment.
 
-The Allowlist Registry maintains a straightforward relationship between these entities:
+### Key Relationship Model
 
-1. Each Ethereum address can be registered for multiple different workloads
-2. Each address has exactly one attestation quote stored for each workloadId it's registered under
-3. The registry tracks whether each (address, workloadId) pair is currently valid or has been marked as outdated
+The Flashtestation Registry maintains a 1:1 mapping between these entities:
 
-This simplified model focuses on tracking the direct relationship between addresses and workloads without the complexity of tracking endorsement bundles.
+1. The TEE-controlled address is the key element in the 1:1 mapping to the parsed attestation struct
```

Comment:
readability-nit: "key element" -> "mapping key".

"is the key element" it's confusing to read the word "key" because I think you're referring to "keypair" or something ECDSA related.